.changeset/happy-dots-sleep.md

```diff
@@ -0,0 +1,8 @@
+---
+"@remix-run/dev": minor
+"@remix-run/react": minor
+"@remix-run/server-runtime": minor
+"@remix-run/testing": minor
```

Comment:
Did `minor` here instead of `patch` since I'm not considering this a change to a current flag but moreso the `deletion` of a current flag and the introduction of a new one.